dali/operators/sequence/optical_flow/optical_flow_adapter/optical_flow_adapter.h

```diff
@@ -26,13 +26,17 @@ namespace optical_flow {
 
 enum struct VectorGridSize {
   UNDEF,
+  SIZE_1 = 2,  /// 1x1 grid
+  SIZE_2 = 3,  /// 2x2 grid
   SIZE_4 = 4,  /// 4x4 grid
+  SIZE_8 = 8,  /// 8x8 grid
   MAX,
 };
```

Comment:
Sorry, but this enum makes no sense and should go away in favour of a plain `int`.
1. The labels are just numbers and their numerical value matches the label (this should be enough, really, to get rid of this).
2. The MAX field is questionable at best in a sparse enumeration, such as this one
3. It doesn't enumerate anything. It _validates_ - but compile-time validation is of little use here.

If the value is supplied at run-time, we'll need run-time validation anyway. As such, we could have a function that checks whether given grid size is valid - but then again, doesn't optical flow API do just that for us?

Reply:
Done